android/build.gradle

```diff
@@ -18,6 +18,7 @@ rootProject.allprojects {
     repositories {
         google()
         jcenter()
+        mavenLocal()
```

Comment:
Not sure we need this, assuming you've left it in

Reply:
Good catch, thanks. That would have been bad. Removing now.